lib/jira/commands/delete.rb

```diff
@@ -0,0 +1,46 @@
+module Jira
+  class CLI < Thor
+
+    desc "delete", "Deletes a ticket in JIRA and the git branch"
+    option :force, :type => :boolean, :default => false
```

Comment:
Can you use ruby 1.9+ hash syntax?
eg. `option :force, type: :boolean, default: false`
I have no immediate plans to provide backwards support for ruby pre-1.9